runelite-client/src/main/java/net/runelite/client/plugins/microbot/thieving/ThievingConfig.java

```diff
@@ -76,11 +76,20 @@ default boolean equipBook() {
     )
     String food = "Food";
 
+    @ConfigItem(
```

Comment:
No need to add an extra config if you do not want to use food. Just change the hitpoints to 0. This will make the script never use food

Reply:
Done this because when there is no food in the inventory it will automatically bank. I can also change it to when food is set to 0 it will skip the banking for food.

Reply:
> Done this because when there is no food in the inventory it will automatically bank. I can also change it to when food is set to 0 it will skip the banking for food. let me know what you think.

Ok fair point